examples/bri-3/components/workgroup/workgroup/workgroupComponent.interface.ts

```diff
@@ -0,0 +1,20 @@
+import { Workgroup } from './workgroup';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Agreement } from '../storage/agreement';
+import { Invitation } from '../invitation/invitation';
+
+export interface IWorkgroupComponent {
+    workgroups: Workgroup[];
+    invitations: Invitation[];
+    archivedWorkgroups: Workgroup[];
+    getWorkgroups(): Workgroup[];
+    createWorkgroup(workgroup: Workgroup): Workgroup;
+    getWorkgroupById(id: string): Workgroup
```

Comment:
@Kasshern not an array?

Reply:
Changed to accept and return an array.  This workgroup Component may be subject to replacement once server framework is implemented, as such it only exists as an interface noting intended functionality.